vocabulary/educational_institution.yml

```diff
@@ -0,0 +1,8 @@
+id: "educational_institution"
+title: "Educational Institution"
+description: ""
+type: "vocabulary"
+properties:
+  name:
+    title: "Name"
+    type: "string"
```

Comment:
Good work, but you described only one field, while your https://github.com/atviriduomenys/manifest/pull/15/files data source has 5. So I think it is better idea to describe all fields, so that #15 pull request would pass.

Also probably it would be better to add it to the same #15 pull request, to see if checks pass, instead of creating new pull request.

Reply:
Understood, I dont know how to merge pull requests so I will make separate ones this time. Following ones will be made as they should be.